build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-beta7'
+        classpath 'com.android.tools.build:gradle:3.1.0-alpha08'
```

Comment:
we good with alphas? don't think we ever talked about policy on this.

Reply:
Actually just let Android Studio update that. I'm fine with it for build tools but either way works for me.

Reply:
I'm in stable channel. I think I'm good with whatever auto updates via stable... think that might be best policy? :man_shrugging: